pkg/delivery/apns.go

```diff
@@ -19,7 +20,14 @@ type ApnsDelivery struct {
 }
 
 func NewApnsDelivery(logger *zap.Logger, opts options.ApnsOptions) (*ApnsDelivery, error) {
-	client, err := getApnsClient([]byte(opts.P8Certificate), opts.KeyId, opts.TeamId)
+	bytes, err := ioutil.ReadFile(opts.P8Certificate)
```

Comment:
This is probably better. I was imagining that someone would just include the entire certificate in the env var/flag. But maybe we want to support both?

Reply:
I could see either as being valid. What about maybe prefixing filenames with `@` like in curl?

Reply:
Might be too cute. What if we keep the existing flag as-is, and add a new option `P8CertificateFilePath`

Reply:
Cool I dig that.